src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,334 @@
+package duke;
+
+/**
+ * Solution below adapted from
+ * https://github.com/nus-cs2113-AY2021S1/contacts/blob/master/src/main/java/Contacts1.java
+ */
+
+import duke.exception.DukeException;
+import duke.exception.TaskDescriptionNotFoundException;
+import duke.exception.TodoDescriptionNotFoundException;
+import duke.exception.DeadlineDescriptionNotFoundException;
+import duke.exception.DeadlineTimeNotFoundException;
+import duke.exception.EventDescriptionNotFoundException;
+import duke.exception.EventTimeNotFoundException;
+import duke.exception.NoTaskInTaskListException;
+import duke.exception.TaskToMarkAsDoneInvalidException;
+import duke.exception.TaskToMarkAsDoneNotFoundException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    private static final int MAX_NUMBER_OF_TASKS = 100;
+    private static Task[] tasks = new Task[MAX_NUMBER_OF_TASKS];
+    private static int taskNumber = 0;
+
+    private static final String inputInstruction = "Please input in the following format:";
+    private static final String printTaskListInputFormat = "  To list all tasks: list";
```

Comment:
constants have a different formatting style i.e. all caps and underscores. Consider changing into that

Reply:
Noted! Thanks for the advice!